ensenso_camera/src/stereo_camera.cpp

```diff
@@ -1143,6 +1144,8 @@ ros::Time StereoCamera::capture() const
 
   NxLibCommand capture(cmdCapture, serial);
   capture.parameters()[itmCameras] = serial;
+  if (captureTimeout > 0)
+    capture.parameters()[itmTimeout] = captureTimeout;
```

Comment:
We always use curly braces, even for one liners.

Reply:
Solved.